docs/experience/webvitals.md

```diff
@@ -0,0 +1,55 @@
+title: Web Vitals
+description: Gathering information about web vitals - essential site health metrics
+
+Supported by Chrome, the [Web Vitals](https://web.dev/vitals/) is a Google initiative to provide unified guidance for metrics that are crucial for delivering the best user experience for the web-based applications. 
+
+<img
+  class="content-modal-image"
+  alt="Web Vitals Overview"
+  src="../../images/experience/webvitals/webvitals.png"
+  title="Web Vitals Overview"
+/>
+
+### Core Web Vitals
+
+Both [Sematext Experience](/experience/) and [Sematext Synthetics](/synthetics/browser-monitor/#web-vitals) support the Core Web Vitals out of the box and include the following metrics:
+
+ * **Largest Contentful Paint (LCP)** - measures *page loading* performance. Google suggests that **LCP** should be **below 2500 milliseconds**, which means that to provide good user experience, the largest contentful paint should occur within 2.5 seconds of when the page first starts loading.
+ * **First Input Delay (FID)** - measures *interactivity* of the web application. Google suggests that to provide good user experience, pages should have a first input delay of **less than 100 milliseconds**.
```

Comment:
FID is not supported by Synthetics, maybe add a small note?

Reply:
Good idea, added.